newsletter/2022-11-29.md

```diff
@@ -1,9 +1,10 @@
-# OSPO Newsletter X
+# OSPO Newsletter 17
 
 
 ## ✨ Monthly Announcements 
 
-* [Monthly Announcement 1]
+* [OSPOCon Japan](https://events.linuxfoundation.org/open-source-summit-japan/) Open Source Summit Japan will take place on the 5th and 6th of December in Yokohama + Virtual.  It is the leading conference in Japan connecting the open source ecosystem under one roof, providing a forum for technologists and open source industry leaders to collaborate and share information, learn about the latest in open source technologies and find out how to gain a competitive advantage by using innovative open solutions.
+
```

Comment:
Since this is a conference, I think OSPOCon Japan fits better in the conferences and CFP space 🙂  Could you please move it to that section?

Reply:
Ok, removed.